src/coreclr/nativeaot/Runtime/ICodeManager.h

```diff
@@ -78,6 +78,47 @@ inline GCRefKind ExtractReg1ReturnKind(GCRefKind returnKind)
     ASSERT(returnKind <= GCRK_LastValid);
     return (GCRefKind)(returnKind >> 2);
 }
+
+#else 
```

Comment:
This can use `#elif defined(TARGET_AMD64)` to make it clear that this block is x64 specific.